python/metatensor-torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -73,6 +74,12 @@ def __init__(
             running, defaults to False.
         :param device: torch device to use for the calculation. If ``None``, we will try
             the options in the model's ``supported_device`` in order.
+        :param properties_to_store: list of model outputs to store as results of the ASE
+            calculator at every step. This is useful when you want to store properties
+            that are not used in the propagation of the dynamics and/or are not standard
+            ASE properties ('energy', 'forces', 'stress', 'stresses', 'dipole',
+            'charges', 'magmom', 'magmoms', 'free_energy', 'energies'). These properties
+            will be available as ``atoms.calc.results['<stored_property>']``.
```

Comment:
Is this typical in ASE? The idea for properties not part of the standard was to use the `run_model` function.

In general, this does not seems to be required for dipole, since ASE already supports them as property.